### ISSUE TITLE
Masking for reflective or transparent objects.

### ISSUE THREAD

**Issue by ejnaren** [alicevision/meshroom#701]
**Is your feature request related to a problem? Please describe.**
When doing photogrammetry models from buildings, windows are often either reflecting from he outside or (semi)transparent from the inside and it really throws the algorithm off in a big way resulting in lots of "shoot out" geometry from these areas.

**Describe the solution you'd like**
I read the upcoming masking feature for adding masking images to tell Meshroom to exclude certain parts of the model by loading in a couple of black/white imegaes describing these areas.
Would the same technique not work for telling Meshroom where there are windows or mirrors and it could exclude/flatten these areas.

**Comment by natowi** [comment]
Yes, you could mask out windows with the upcomming masking feature. You could also try using a polarization filter for your camera.

https://github.com/alicevision/meshroom/issues/188

https://github.com/alicevision/meshroom/issues/566

**Comment by natowi** [comment]
https://github.com/alicevision/Meshroom/wiki/New-features-in-Meshroom-2025.1
... and more to come
